src/res/homepage/ES/manual-es.md

```diff
@@ -1,209 +1,5 @@
-
-### Canales de Audio
-
-![Audio channels](audiochannels.png)
-
-Selecciona el número de canales de audio a utilizar para la comunicación entre cliente y servidor. Hay tres modos disponibles:
-
-Los modos Mono y Estéreo utilizan uno y dos canales de audio respectivamente.
-
-Entrada-mono/Salida-estéreo: La señal de audio enviada al servidor es mono pero la señal que vuelve es estéreo. Esto es útil si la tarjeta de audio tiene un instrumento en un canal de entrada y un micrófono en el otro. En ese caso las dos señales de entrada pueden combinarse en un canal mono pero la mezcla del servidor se escucha en estéreo.
-
-Activar el modo estéreo aumentará la tasa de envío de datos. Asegúrate de que la tasa de subida no excede el ancho de banda disponible en tu conexión a internet.
-
-En el caso del modo estéreo, no estará disponible la selección de canal para el efecto de reverb en la ventana principal puesto que en este caso el efecto se aplicará a ambos canales.
-
-### Calidad del Audio
-
-![Audio quality](audioquality.png)
-
-Cuanto mayor la calidad del audio, mayor la tasa de subida del audio. Asegúrate de que tu tasa de subida no excede el ancho de banda de tu conexión a Internet.
-
-### Nivel cliente nuevo
-
-![New client level](newclientlevel.png)
-
-Este ajuste define el nivel del fader de una nueva conexión de cliente, en porcentaje. Si se conecta un nuevo cliente al servidor actual, el nivel inicial de su fader tomará este valor si no se ha especificado anteriormente un valor para ese cliente de una conexión anterior.
-
-### Interfaz oscura
-
-![Fancy skin](fancyskin.png)
-
-Esto aplicará un aspecto oscuro a la ventana principal.
-
-### Dirección personalizada del servidor central
-
-![Central server address](centralserveraddress.png)
-
-Deja esto en blanco a menos que necesites escribir la dirección de un servidor distinto a los que hay por defecto.
-
-### Parámetro de estado de conexión actual
-
-![Indicators](indicators.png)
-
-El Ping es el tiempo que requiere el flujo de audio para viajar desde el cliente al servidor y volver. Este retardo lo determina la red y debería ser de unos 20-30 ms. Si este retardo es mayor de unos 50 ms, la distancia al servidor es demasiado grande o tu conexión a internet no es óptima.
-
-El retardo total se calcula con el ping y el retardo ocasionado por la configuración de buffers.
-
-La Tasa de Subida de Audio depende del tamaño actual de los paquetes de audio y la configuración de compresión de audio. Asegúrate de que la tasa de subida no es mayor que la velocidad de subida disponible (comprueba la tasa de subida de tu conexión a internet, por ejemplo con [speedtest.net](http://speedtest.net)).
+## Movido a https://jamulus.io/wiki/Software-Manual
```

Comment:
I'd link  https://jamulus.io/es/wiki/Software-Manual

Reply:
OK I think I've done that (not sure).